test/testFunction.php

```diff
@@ -0,0 +1,38 @@
+<?php
+require_once '../function.php';
```

Comment:
Missing newline between opening PHP tag and require_once

Reply:
new line added